homeassistant/components/frontier_silicon/media_player.py

```diff
@@ -39,7 +39,16 @@ async def async_setup_entry(
 
     afsapi: AFSAPI = hass.data[DOMAIN][config_entry.entry_id]
 
-    async_add_entities([AFSAPIDevice(config_entry.title, afsapi)], True)
+    async_add_entities(
+        [
+            AFSAPIDevice(
+                config_entry.unique_id or config_entry.entry_id,
```

Comment:
The unique ID of a config entry is not compatible with the unique ID of the entities (different rules).

I would recommend just to use the `entry_id` in this case.